packages/core/src/components/ui/ProductGallery/ProductGallery.tsx

```diff
@@ -67,6 +67,19 @@ export interface ProductGalleryProps {
   }
   sortBySelector?: SortProps
   productCard?: Pick<ProductCardProps, 'showDiscountBadge' | 'bordered'>
+  emptyGallery?: {
```

Comment:
Let's move this to `EmptyGallery` component as `EmptyGalleryProps` interface and import here.